sources/Outputs/src/Dyd.cpp

```diff
@@ -206,12 +206,10 @@ Dyd::writeGenerator(const algo::GeneratorDefinition& def, const std::string& bas
 }
 
 std::vector<boost::shared_ptr<dynamicdata::BlackBoxModel>>
-Dyd::writeConstantsModel(const std::string& basename) {
+Dyd::writeConstantsModel() {
   std::vector<boost::shared_ptr<dynamicdata::BlackBoxModel>> ret;
   auto model = dynamicdata::BlackBoxModelFactory::newModel(signalNModelName_);
   model->setLib("SignalN");
-  model->setParFile(basename + ".par");
-  model->setParId(constants::signalNParId);
```

Comment:
signalNParId is not used anymore. remove it

Reply:
Thanks, done